types/ember-data/test/model.ts

```diff
@@ -14,9 +14,13 @@ const Person = DS.Model.extend({
         return `${this.get('firstName')} ${this.get('lastName')}`;
     }),
 
-    point: DS.attr('point', { defaultValue: () => Point.create({ x: 1, y: 2 })}),
+    pointOne: DS.attr('point', { defaultValue: () => Point.create({ x: 1, y: 2 })}),
+    pointTwo: DS.attr('point', { defaultValue: Point.create({ x: 1, y: 2 }) })
```

Comment:
Setting the value without a callback was not allowed previously.

Reply:
Can you reference https://api.emberjs.com/ember-data/3.24/functions/@ember-data%2Fmodel/attr in the PR description pointing out that `defaultValue` can be either a function or a value (for those that come across this PR later)?

Reply:
This is [not actually allowed](https://github.com/emberjs/data/blob/cafc3db253d1661015a7335a51c2dd0bd8b7dd86/packages/model/addon/-private/attr.js#L19-L22).

Reply:
Ah, interesting. I had missed that. Interestingly, the docs seems to be wrong here:

> defaultValue: Pass a string or a function to be called to set the attribute to a default value if and only if the key is absent from the payload response.

The example uses `false` which is neither of the above but should be allowed.

Reply:
Yeah, should say something like a primitive or a function.